web/client/components/buttons/ZoomToMaxExtentButton.jsx

```diff
@@ -103,6 +106,12 @@ var ZoomToMaxExtentButton = React.createClass({
         // adapt the map view by calling the corresponding action
         this.props.changeMapView(newCenter, newZoom,
             this.props.mapConfig.bbox, this.props.mapConfig.size, null, this.props.mapConfig.projection);
+    },
+    zoomToInitialExtend() {
+        // zooming to the initial extend based on initial map configuration
+        var mapConfig = this.props.mapInitialConfig;
+        this.props.changeMapView(mapConfig.center, mapConfig.zoom,
+            mapConfig.bbox, mapConfig.size, null, mapConfig.projection);
```

Comment:
I don't think you have bbox in initial config, neither size, you will probably need to calculate bbox from center, zoom and projection, and get map size from current map.